### PR TITLE
added 2FA notes

### PR DESCRIPTION
**Pull request by himorin**

### CODE REVIEW

File: services/account.rst [hunk 1]
```diff
@@ -120,3 +123,13 @@ links.
   and find 'change password' link. 
   If you cannot find, you are under a single sign on. 
 * `Sumire-PFS slack account settings <https://sumire-pfs.slack.com/account/settings#password>`_
+
+If you have issues on 2FA
```
Comment (yukimoritani) [inline services/account.rst]: This paragraph seems not related to **"Lost Account or Change password"** Section.
How does it sound to 
- Add "Help" section, and
- Have "lost account or change password" and "2FA" subsections?

**Review by yukimoritani — COMMENTED** [review]
Just a thought..

**Comment by himorin** [comment]
@yukimoritani thx. updated.

**Review by yukimoritani — CHANGES_REQUESTED** [review]
Thank you for consideration.

**Review by yukimoritani — APPROVED** [review]

**Comment by yukimoritani** [comment]
Thanks for updates.
I'm sorry... my fingers had slipped...
